packages/flutter/lib/src/material/tabs.dart

```diff
@@ -822,14 +822,25 @@ class _TabBarViewState extends State<TabBarView> {
 /// ancestor.
 class TabPageSelector extends StatelessWidget {
   /// Creates a compact widget that indicates which tab has been selected.
-  TabPageSelector({ Key key, this.controller }) : super(key: key);
+  TabPageSelector({
+    Key key,
+    this.controller,
+    this.emptyIndicatorFillColor: Colors.transparent,
+    this.indicatorSize: 12.0
+  }) : super(key: key);
 
   /// This widget's selection and animation state.
   ///
   /// If [TabController] is not provided, then the value of [DefaultTabController.of]
   /// will be used.
   final TabController controller;
 
+  /// The fill color for an unselected indicator. Defaults to Colors.transparent.
```

Comment:
You should add more color to these docs in a second paragraph, saying why you would set it.